spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/cql/Schema.scala

```diff
@@ -148,6 +155,12 @@ case class TableDef(
        |  PRIMARY KEY ($primaryKeyClause)
        |)""".stripMargin
   }
+
+  type ValueRepr = CassandraRow
+  
+  def newInstance(columnValues: Any*): CassandraRow = {
```

Comment:
Can't be on CassandraRow, because columnNames information is missing. And there is already an `apply` on CassandraRow taking the columnNames and columnValues.